app/backend/app/externalAPI/geocode.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import geojson
+import aiohttp
+import asyncio
+from geocodio import GeocodioClient
+
+from dotenv import load_dotenv
+import os
+
+from app.backend.utility.coordinates import Point, Segment, Grid
+
+
+class GeocodioRequests:
+    def __init__(self) -> None:
+        self.__key = self.__load_key()
+        self.client = GeocodioClient(self.__key)
+
+    def __load_key(self) -> str:
+        load_dotenv()
+        return os.getenv("GEOCODE_API_KEY")
+
+    async def reverse_geocode_async(self, grid:Grid):
+        url = f"https://api.geocod.io/v1.6/reverse"
```

Comment:
it is better to define the URL as a constant variable, especially if it is a fixed value that does not change during the execution of the program)

Reply:
will do